### PR TITLE
chore: add issue templates for bug report and feature request

### LINKED ISSUE #14: Lab issue template
Migrating issue to this repo

> More or less already written out in the lab notebook, but we could implement it via the github issue template system: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

_Originally posted by @jsstevenson  in https://github.com/GenomicMedLab/wagner-lab-issues/issues/20_

> I think we should either have the developers meet for this or we make an initial template here or in lab-knowledgebase and make a PR so we can discuss. 
> 
> I assume we'd want one for bug + feature request? Do we want other templates?  
> 
> Honestly, it might be nice to use this repo or make a separate private repo where other lab members can post questions / issues to developers (in cases where we wouldn't want to make an issue in the project repo). I'm just leaning towards this because I'm not sure if people would follow what's in the templates when slacking 

_Originally posted by @korikuzma in https://github.com/GenomicMedLab/wagner-lab-issues/issues/20#issuecomment-1364281936_

### PR DESCRIPTION
**Pull request by korikuzma**
close #14

### CODE REVIEW

**Review by jsstevenson — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: Provide a clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Pprovide detailed steps to replicate the bug.
```
Comment (MichaelMcNCH) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:14]: "Pprovide" -> "Provide"

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      placeholder: |
+        1. In this environment...
```
Comment (MichaelMcNCH) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:16]: Four instances of "1.", assuming this should be 1,2,3,4?

Comment (korikuzma) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:16]: Ah was using markdown conventions seen [here](https://www.markdownguide.org/basic-syntax/#ordered-lists), but it doesn't appear to work with GitHub issue templates. Will change!

**Review by MichaelMcNCH — APPROVED** [review]

**Comment by korikuzma** [comment]
Going to update to add cause / fix

**Review by MichaelMcNCH — APPROVED** [review]

**Review by jsstevenson — APPROVED** [review]

**Review by katiestahl — APPROVED** [review]

**Pull request merged**